src/main/java/com/palantir/docker/compose/execution/RetryingDockerCompose.java

```diff
@@ -64,6 +64,15 @@ public void rm() throws IOException, InterruptedException {
     }
 
     @Override
+    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName,
```

Comment:
Agreed - I say we make this a non-retryable one for now and see what happens.

Also, if this gets any more calls, I'm totally making this use annotations and some fancy reflection nonesense.